lib/src/main/java/com/diffplug/spotless/sql/HibernateStep.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2016 DiffPlug
```

Comment:
...but we should probably change this to "2017". :)

Reply:
2018 is coming soon enough, I was gonna change our copyright notice then :)  Also want to change the notice to point to https://github.com/diffplug/spotless/graphs/contributors

Reply:
Sounds very sensible to me. :)